### PR TITLE
Adding the crop code made for videos to images

### PR DESCRIPTION
**Pull request by THE-COB**
https://github.com/nerfstudio-project/nerfstudio/issues/1595

### CODE REVIEW

File: nerfstudio/process_data/process_data_utils.py [hunk 1]
```diff
@@ -174,7 +174,11 @@ def convert_video_to_images(
 
 
 def copy_images_list(
-    image_paths: List[Path], image_dir: Path, crop_border_pixels: Optional[int] = None, verbose: bool = False
+    image_paths: List[Path],
+    image_dir: Path,
+    crop_border_pixels: Optional[int] = None,
+    crop_factor: Tuple[float, float, float, float] = (0.0, 0.0, 0.0, 0.0),
```
Comment (tancik) [inline nerfstudio/process_data/process_data_utils.py:180]: Add `crop_factor` to docstring

File: nerfstudio/process_data/process_data_utils.py [hunk 2]
```diff
@@ -266,7 +282,9 @@ def copy_and_upscale_polycam_depth_maps_list(
     return copied_depth_map_paths
 
 
-def copy_images(data: Path, image_dir: Path, verbose) -> OrderedDict[Path, Path]:
+def copy_images(
+    data: Path, image_dir: Path, verbose, crop_factor: Tuple[float, float, float, float] = (0.0, 0.0, 0.0, 0.0)
```
Comment (tancik) [inline nerfstudio/process_data/process_data_utils.py:286]: Add `crop_factor` to docstring

**Review by tancik — APPROVED** [review]
LGTM

**Comment by smallfly** [comment]
Thanks!

**Pull request merged**